### PR TITLE
feat: contracts addressbook and on-chain guide

### PR DESCRIPTION
**Pull request by 0xPenryn**
and various other smaller fixes

in draft until all addresses and ENS names are available

### CODE REVIEW

File: src/pages/reference/contracts.mdx [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    <tr>
+      <th>Contract</th>
+      <th>Optimism Mainnet</th>
+      <th>Optimism Goerli Testnet</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td className="align-middle">World ID Router</td>
+      <td className="align-middle">[`optimism.id.worldcoin.eth`](https://goerli-optimism.etherscan.io/address/0x57f928158C3EE7CDad1e4D8642503c4D0201f611#code)</td>
```
[the remark this answers is not in the document]
Comment (0xPenryn) [reply src/pages/reference/contracts.mdx]: fixed

File: src/pages/reference/contracts.mdx [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+      <td className="align-middle">[`goerli.id.worldcoin.eth`](https://goerli.etherscan.io/address/0x05C4AE6bC33e6308004a47EbFa99E5Abb4133f86#code)</td>
+    </tr>
+    <tr>
+      <td className="align-middle">Orb State Bridge</td>
+      <td className="align-middle">[`0xa3d93b74214b80760288091675688e1ead7838f2`](https://etherscan.io/address/0xa3d93b74214b80760288091675688e1ead7838f2#code)</td>
+      <td className="align-middle">[`0xcd60da00e20be3d23e2e135283423fd86867c98d`](https://goerli.etherscan.io/address/0xcd60da00e20be3d23e2e135283423fd86867c98d#code)</td>
+    </tr>
+    <tr>
+      <td className="align-middle">Orb Identity Manager</td>
+      <td className="align-middle">[`0x316350d3ec608ffc30b01dcb7475de1c676ce910`](https://etherscan.io/address/0x316350d3ec608ffc30b01dcb7475de1c676ce910#code)</td>
```
Comment (Dzejkop) [inline src/pages/reference/contracts.mdx:112]:
```suggestion
      <td className="align-middle">[`0x316350d3ec608ffc30b01dcb7475de1c676ce910`](https://etherscan.io/address/0xf7134CE138832c1456F2a91D64621eE90c2bddEa#code)</td>
```

Comment (Dzejkop) [reply src/pages/reference/contracts.mdx:112]: This changed some time ago and I messed up the change in Notion addressbook. Should have been fixed by now though

File: src/pages/advanced/custom-on-chain.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+
```
[the remark this answers is not in the document]
Comment (0xPenryn) [reply src/pages/advanced/custom-on-chain.mdx:1]: For more information on use cases that require more complex setups (such as multiple actions, other types of signals, or multiple credential types), see the [Advanced On-Chain Verification](/advanced/custom-on-chain) page.

^^ some context on what that page will be for

File: src/pages/reference/contracts.mdx [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+  </thead>
+  <tbody>
+    <tr>
+      <td className="align-middle">World ID Router</td>
+      <td className="align-middle">[`id.worldcoin.eth`](https://etherscan.io/address/0x163b09b4fe21177c455d850bd815b6d583732432#code)</td>
+      <td className="align-middle">[`goerli.id.worldcoin.eth`](https://goerli.etherscan.io/address/0x05C4AE6bC33e6308004a47EbFa99E5Abb4133f86#code)</td>
+    </tr>
+    <tr>
+      <td className="align-middle">Orb State Bridge</td>
+      <td className="align-middle">[`0xa3d93b74214b80760288091675688e1ead7838f2`](https://etherscan.io/address/0xa3d93b74214b80760288091675688e1ead7838f2#code)</td>
```
Comment (Dzejkop) [inline src/pages/reference/contracts.mdx:107]:
```suggestion
      <td className="align-middle">[`0x86D26Ed31556EA7694BD0cC4e674D7526f70511a`](https://etherscan.io/address/0x86D26Ed31556EA7694BD0cC4e674D7526f70511a#code)</td>
```

File: src/pages/reference/contracts.mdx [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+  </thead>
+  <tbody>
+    <tr>
+      <td className="align-middle">World ID Router</td>
+      <td className="align-middle">[`optimism.id.worldcoin.eth`](https://optimistic.etherscan.io/address/0x57f928158C3EE7CDad1e4D8642503c4D0201f611#code)</td>
+      <td className="align-middle">[`op-goerli.id.worldcoin.eth`](https://goerli-optimism.etherscan.io/address/0xfc1315089316fcfe586a8e0a92873c258de8aac1#code)</td>
+    </tr>
+    <tr>
+      <td className="align-middle">Orb State Bridge</td>
+      <td className="align-middle">[`0x4055b6d4018e92e4d000865e61e87b57a4e5ab49`](https://optimistic.etherscan.io/address/0x4055b6d4018e92e4d000865e61e87b57a4e5ab49#code)</td>
```
Comment (Dzejkop) [inline src/pages/reference/contracts.mdx:145]:
```suggestion
      <td className="align-middle">[`0x42FF98C4E85212a5D31358ACbFe76a621b50fC02`](https://optimistic.etherscan.io/address/0x42FF98C4E85212a5D31358ACbFe76a621b50fC02#code)</td>
```

File: src/pages/reference/contracts.mdx [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+  </thead>
+  <tbody>
+    <tr>
+      <td className="align-middle">World ID Router</td>
+      <td className="align-middle">[`polygon.id.worldcoin.eth`](https://polygonscan.com/address/0x515f06B36E6D3b707eAecBdeD18d8B384944c87f#code)</td>
+      <td className="align-middle">[`mumbai.id.worldcoin.eth`](https://mumbai.polygonscan.com/address/0x719683F13Eeea7D84fCBa5d7d17Bf82e03E3d260#code)</td>
+    </tr>
+    <tr>
+      <td className="align-middle">Orb State Bridge</td>
+      <td className="align-middle">[`0x68cab22b7cef23f8cce39aff9b13aa2e6aba8aff`](https://polygonscan.com/address/0x68cab22b7cef23f8cce39aff9b13aa2e6aba8aff#code)</td>
```
Comment (Dzejkop) [inline src/pages/reference/contracts.mdx:173]:
```suggestion
      <td className="align-middle">[`0x2Ad412A1dF96434Eed0779D2dB4A8694a06132f8`](https://polygonscan.com/address/0x2Ad412A1dF96434Eed0779D2dB4A8694a06132f8#code)</td>
```

File: src/pages/_app.tsx [hunk 7]
```diff
@@ -67,7 +67,7 @@ const App: FC<AppProps> = ({ Component, pageProps }) => {
 				<link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png" />
 				<link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png" />
 			</Head>
-			{process.env.NODE_ENV === 'production' && <Clippy theme="light" />}
+			{/* {process.env.NODE_ENV === 'production' && <Clippy theme="light" />} */}
```
[the remark this answers is not in the document]
Comment (0xPenryn) [reply src/pages/_app.tsx:70]: search does work (even running locally!), but it's either trained on or reading from (cc: @m1guelpf) the current live docs, so it needs a look

File: src/pages/advanced/sign-in-widget.mdx [hunk 8]
```diff
@@ -9,7 +9,7 @@ It's also possible to implement Sign In with Worldcoin using IDKit. See our [OID
 
 ## How it works
 
-The Sign In widget is a wrapper around the [IDKit Widget](/reference/idkit/widget) that implements the OIDC imlpicit flow for Sign In with Worldcoin. It is a modal that opens when the user clicks the Sign In button.
+The Sign In widget is a wrapper around the [IDKit Widget](/reference/idkit/widget) that implements the OIDC impicit flow for Sign In with Worldcoin. It is a modal that opens when the user clicks the Sign In button.
```
[the remark this answers is not in the document]
Comment (0xPenryn) [reply src/pages/advanced/sign-in-widget.mdx]: fixed... for real this time

**Pull request merged**